docker/ravendb-ubuntu/get-server-url.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# This script takes scheme as first arg (http, https, tcp etc.)
+
+[[ $# -lt 1 ]] && scheme="http" || scheme=$1
```

Comment:
can we put get-server-url and get-server-var stuff in a single `serverutils.sh` file?
One less file to worry about and if we wanted to add something it's just a function away.

Reply:
done